Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -72,12 +72,14 @@ void ADD_MATCH(R_DECOMP &match, int rlabel) {
 
 RGroupDecomposition::RGroupDecomposition(
     const ROMol &inputCore, const RGroupDecompositionParameters &params)
-    : data(new RGroupDecompData(inputCore, params)) {}
+    : data(new RGroupDecompData(inputCore, params)) {
+}
 
 RGroupDecomposition::RGroupDecomposition(
     const std::vector<ROMOL_SPTR> &cores,
     const RGroupDecompositionParameters &params)
-    : data(new RGroupDecompData(cores, params)) {}
+    : data(new RGroupDecompData(cores, params)) {
+}
 
```

Comment:
I'm not clear why this formatting change is here. When I run clang-format across the code with the current RDKit settings, the existing form is preferred.

Reply:
I guess I forgot to format the file.  I've made the change